cmd/vic-machine/create/create.go

```diff
@@ -466,6 +466,12 @@ func (c *Create) Flags() []cli.Flag {
 			Usage:       "Time to wait for create",
 			Destination: &c.Timeout,
 		},
+		cli.BoolFlag{
+			Name:        "asymmetric-routes, ar",
+			Usage:       "Set up the Virtual Container Host for asymmetric routing",
```

Comment:
I know its cryptic, but I think adding more here will just add more confusion. @hickeng has already requested @stuclem document this in https://github.com/vmware/vic/issues/4613#issuecomment-296783581, which is where most of the detail would go.